package.json

```diff
@@ -31,7 +31,7 @@
     "relay": "relay-compiler --src ./src --schema data/schema.graphql --extensions ts tsx",
     "sync-colors": "cd externals/elan && git fetch && git checkout origin/master && cp components/lib/variables/colors.json ../../data",
     "sync-externals": "npm run-script sync-schema && npm run-script sync-colors",
-    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
+    "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/use_unaltered_text_field && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
```

Comment:
temp to point to that new schema

Reply:
When you have a branch checked out locally, which you probably do in this case, you can just run the `dump-schema.js` script _from_ that checkout.

```
$ cd metaphysics
$ yarn dump-schema -- ../emission/data
```

Reply:
oo gonna try this now!